server/src/main/java/org/spine3/server/entity/storage/Column.java

```diff
@@ -42,8 +42,8 @@
  *
  *
  * <p>These methods represent the Columns:
- * <code>
- *      <pre>
+ * <pre>
+ *      <code>
```

Comment:
Please prefer `{@code ... }` to `<code> ... </code>` here and in other similar places.